setup.py

```diff
@@ -31,14 +31,14 @@
 ]
 
 setup(
-    name='tensornetwork',
+    name='TensorNetwork',
     version=__version__,
     url='http://github.com/google/TensorNetwork',
-    author='The TensorNetwork Developers',
-    author_email=' chaseriley@google.com',
+    author='TensorNetwork Developers',
```

Comment:
Do not remove the "The"

Reply:
We'll have to pick one change this to either "The TensorNetwork Developers" or "The TensorNetwork Authors". I see either of these being used and places.